packages/web-app/src/modules/notifications/utils.tsx

```diff
@@ -57,20 +57,26 @@ export const getNormalizedNovuNotification = (novuNotification: IMessage): Notif
       return { action: getNormalizedNotificationAction(key, cta[key]) }
     })
 
-    const trackId = v1.type === 'info' ? NotificationMessageCategory.NovuInfo : NotificationMessageCategory.NovuWarning
+    let variant: NotificationVariant = 'news'
+    if (v1.type === 'info') {
+      variant = 'news'
+    } else {
+      variant = v1.type
+    }
 
     return {
-      trackId,
       novuId: novuNotification._id,
       title: v1.title,
       body: v1.body,
       read: novuNotification.read,
       seen: novuNotification.seen,
       createDate: new Date(novuNotification.createdAt),
       acknowledged: novuNotification.read,
+      badgeUrl: v1.badgeUrl,
```

Comment:
`const { cta, title, body, badgeUrl } = v1`
```suggestion
      badgeUrl: badgeUrl,
```